Cargo.toml

```diff
@@ -2,7 +2,7 @@
 description = "OpenEthereum"
 name = "openethereum"
 # NOTE Make sure to update util/version/Cargo.toml as well
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
has 3.0.0 been published already?

Reply:
no

Reply:
I'm not a big fan of re-pushing a tag/image, but I understand if you want to maintain the number for monday...
Will remove this then